### ISSUE TITLE
Workers can no longer improve outside city boundary

### ISSUE THREAD

**Issue by L0j1k** [yairm210/Unciv#3053]
I just upgraded to the latest version (3.10.2 it claims) this morning and started a new game and my workers are now no longer able to make improvements on the other side of the city boundary. I realize there are two (other)[https://github.com/yairm210/Unciv/issues/2818] (issues)[https://github.com/yairm210/Unciv/issues/2778] complaining about this being a form of exploitation to weaken enemy AI and can see (the change)[https://github.com/yairm210/Unciv/pull/2957] made to address these, but this really destroyed my play style.

I downgraded the game to the last version to avoid this because I agree that it is an artificial limitation and not fun to have to pass over a bunch of idle workers for a bunch of turns. This change was too far in the other direction. Maybe a worker can refuse to build within X tiles of a non-allied city, or AI could perceive non-allied improvements (or modification of its own improvements by a non-allied actor) as hostile, or something.

**Comment by L0j1k** [comment]
Haha well Github's UI has been broken for me since their truly amazing update and I guess for some reason it won't let me edit this issue to fix my stupid links. :)

**Comment by L0j1k** [comment]
Haha well Github's UI has been broken for me since their truly amazing update and I guess for some reason it won't let me edit this issue to fix my stupid links. :)

**Comment by L0j1k** [comment]
Haha well Github's UI has been broken for me since their truly amazing update and I guess for some reason it won't let me edit this issue to fix my stupid links. :)

**Comment by L0j1k** [comment]
Haha well Github's UI has been broken for me since their truly amazing update and I guess for some reason it won't let me edit this issue to fix my stupid links. :)

**Comment by TheRexYo** [comment]
At the top right edge of your first post, is the "more options" ellipsis there? It should be under that.

**Comment by L0j1k** [comment]
It is, but the UI is fucked, won't open the edit modal for editing.

**Comment by TheRexYo** [comment]
Weird. Not much I can say beyond that. 
Anyway, you're not wrong. Plus, I think the changes they made actually prevent great generals from building citadels nearby enemy cities. (not that it's an intended feature, but it was a useful means of keeping a sustained assault without losing tons of soldiers).

**Comment by TheRexYo** [comment]
Also, the links you posted aren't broken on the discord, oddly.

**Comment by ravignir** [comment]
That was changed because Unciv is a Civ5 clone and Workers can't build improvements outside your borders (except fort/road\railroad) in civ5

**Comment by L0j1k** [comment]
That may be, but as mentioned in the other issues, it's an artificial limitation and makes the game grindy and unfun for my play style. I opened the issue because I'm sure the purist approach of cloning bugs from Civ5 is also not a design goal, and so it's worth discussing an alternative to cloning this particular behavior because of how it negatively affects gameplay and balance.

**Comment by HadeanLake** [comment]
[Improvements_Everywhere.zip](https://github.com/yairm210/Unciv/files/5146505/Improvements_Everywhere.zip)
Those who think it negatively affects gameplay and balance can fix it with a mod

**Comment by yairm210** [comment]
As with all slightly controversial issues, I opened a poll on our Discord server 
We'll see what the community days, but there will only be one final configuration.

**Comment by yairm210** [comment]
The People have Spoken
The vast majority voted for "Your territory and neutral tiles" above "Your territory only" and "Everywhere"
So that's what it's going to be

**Comment by L0j1k** [comment]
Aloha @yairm210! My apologies for pinging you on a closed issue but I figured I'd add to this instead of opening a new one since it's the same change that made its way back into release. I updated Unciv a couple of days ago to version 3.12.10 (from f-droid) and my workers are no longer able to improve tiles outside city boundaries. I looked in the changelog and it seems this was reverted in version 3.11.10. Just wondering if your decision here a while ago slipped your mind? If it was a deliberate change, could this feature be made into a configuration setting? Maybe I'm boring :) but I still rely on my far-flung workers to be a step ahead of my cities. Thanks! (Also unrelated but f-droid links to your Patreon, however it doesn't appear possible to actually become a patron)
